conda-recipes/scvi/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "scvi" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ca87155662d85605f86c5e86b7b9f64d881b882177b9642fff8f0ea215c6cb1a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
```

Comment:
What's the difference between `host` and `run`?

Reply:
`host` represents packages that need to be specific to the target platform when the target platform is not necessarily the same as the native build platform. `run` include packages required to run the package. (They are automatically generated when I build conda package from pip)

Reply:
I don't see host used in other people's conda recipes, so it might not be something we want. Maybe better to omit it and see if that creates problems?

Reply:
Yup 😊 I make these changes and pushed to `yl/conda`.